legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/auth/GitLabTokenHelper.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.gitlab.auth;
+
+import org.finos.legend.sdlc.server.auth.Token;
+import org.finos.legend.sdlc.server.gitlab.GitLabAppInfo;
+import org.gitlab4j.api.Constants;
+
+public class GitLabTokenHelper
+{
+    public static Token.TokenBuilder encodeGitLabToken(Token.TokenBuilder builder, GitLabToken token, GitLabAppInfo appInfo)
+    {
+        if (token != null)
```

Comment:
See the below comment about maintaining backward compatibility. Perhaps this should start with an integer indicating how many (1 or 0) tokens are being encoded.